setup.cfg

```diff
@@ -39,3 +39,4 @@ openstack.esiclient.v1 =
     esi_trunk_list = esiclient.v1.trunk:List
     esi_trunk_add_network = esiclient.v1.trunk:AddNetwork
     esi_trunk_remove_network = esiclient.v1.trunk:RemoveNetwork
+
```

Comment:
This looks like an erroneous change.

Reply:
Sorry, I forgot to add the new entry points in it...